GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -101,6 +101,9 @@ public FormResolveConflicts(GitUICommands commands, bool offerCommit = true)
             Translate();
             _offerCommit = offerCommit;
             _fullPathResolver = new FullPathResolver(() => Module.WorkingDir);
+
+            FileName.DataPropertyName = nameof(ConflictData.Filename);
+            authorDataGridViewTextBoxColumn1.DataPropertyName = "Author"; // TODO this property does not exist on the target type
```

Comment:
This is one of two possible bugs. I couldn't work out what this column was for. Might be a candidate for removal.